yarn-project/circuits.js/src/structs/rollup/root_rollup.ts

```diff
@@ -120,6 +120,43 @@ export class RootRollupPublicInputs {
     return new RootRollupPublicInputs(...RootRollupPublicInputs.getFields(fields));
   }
 
+  static makeFake() {
```

Comment:
Let's rename to `empty()`

Reply:
Was my original name, but used makefake as it cointained the aggregation object that was named makefake so assumed there was a reason the underlying was not called empty and that it should convey that as well.